hello-nextflow/hello-channels.nf

```diff
@@ -8,24 +19,13 @@ process sayHello {
     publishDir 'results', mode: 'copy'
 
     input:
-        val greeting
+    val greeting
 
     output:
-        path 'output.txt'
+    path 'output.txt'
 
     script:
     """
-    echo '$greeting' > output.txt
+    echo '${greeting}' > output.txt
```

Comment:
are the curly braces going to be mandatory? why do we need them?

Reply:
They are optional in certain cases like simple variable names. The formatter doesn't try to be smart here, it just always uses braces

Reply:
As for whether we should put them everywhere, there is a trade-off between consistency and shortcuts

Using curly braces everywhere requires a bit more typing but makes the code more consistent, therefore easier to read

Omitting the curly braces where possible saves a bit on typing but makes the code less consistent, thus the reader has to work a bit more to understand the different syntax forms

So I think it comes down to how much time we spend reading vs writing code. At least in software engineering, we spend way more time reading code, so I tend to favor consistency

Reply:
I think it's easier to teach consistency as well. So I like this change.